tileview/src/main/java/com/qozix/tileview/tiles/Tile.java

```diff
@@ -255,7 +258,12 @@ void reset() {
     mState = State.UNASSIGNED;
     mRenderTimeStamp = null;
     if( mBitmap != null && !mBitmap.isRecycled() ) {
-      mBitmap.recycle();
+      BitmapRecycler recycler = mBitmapRecyclerReference.get();
```

Comment:
as mentioned earlier, i'd probably move the `isRecycled` check from here to the implementation